features/support/env.rb

```diff
@@ -6,15 +6,15 @@
 # Install latest versions of bugsnag-android
   run_required_commands([
     [
-      "./gradlew", "assembleRelease", "publishToMavenLocal", "-x", "lintVitalRelease"
```

Comment:
This task is no longer in the project after the example app restructure, so needs to be removed to make mazerunner run.

Reply:
Should it have been removed in the the example app restructure PR then?

Reply:
Ideally yes - this is only needed to run the local version of mazerunner, which is why it was originally missed.